plugins/org.locationtech.udig.catalog.wms/src/org/locationtech/udig/catalog/wms/preferences/WmsPreferencePage.java

```diff
@@ -0,0 +1,39 @@
+package org.locationtech.udig.catalog.wms.preferences;
+
+import org.locationtech.udig.catalog.internal.wms.WmsPlugin;
+
+import org.eclipse.jface.preference.FieldEditorPreferencePage;
+import org.eclipse.jface.preference.IntegerFieldEditor;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchPreferencePage;
+
+
+/**
+ * 
+ * @author Nikolaos Pringouris <nprigour@gmail.com>
+ *
+ */
+public class WmsPreferencePage extends FieldEditorPreferencePage
+implements IWorkbenchPreferencePage {
+
+	public static final String WMS_RESPONSE_TIMEOUT = "WMS_RESPONSE_TIMEOUT";
+	
+	public WmsPreferencePage() {
+		super(GRID);
+	}
+
+
+	@Override
+	public void init(IWorkbench workbench) {
+		setPreferenceStore(WmsPlugin.getDefault().getPreferenceStore());
+		setDescription("Configuration Options about WMS connections");
+	}
+
+	@Override
+	protected void createFieldEditors() {
+		
+		addField(new IntegerFieldEditor(WMS_RESPONSE_TIMEOUT, "WMS response timeout", getFieldEditorParent()));   

```

Comment:
Could you please externalize the string to properties files as well? So Users could translated it usin e.g. Transifex (https://www.transifex.com/udig/udig/dashboard/)